### ISSUE TITLE
CrossEntropy 文档可能错误

### ISSUE THREAD

**Issue by morgan-bc** [PaddlePaddle/Paddle#31705]
### 文档问题描述：
`paddle.nn.CrossEntropyLoss` 和 `paddle.nn.functional.cross_entropy` 二者文档存在相互矛盾的地方。
`paddle.nn.CrossEntropyLoss` 对输入的要求为

> input (Tensor): - 输入 Tensor ，数据类型为float32或float64。**其形状为 [N,C] , 其中 C 为类别数**。对于多维度的情形下，它的形状为 [N,C,d1,d2,...,dk] ，k >= 1。

 `paddle.nn.functional.cross_entropy`  对输入的要求为

> input (Tensor) – 维度为 [N1,N2,...,Nk,D] 的多维Tensor，**其中最后一维D是类别数目**。数据类型为float32或float64。

但实际上 `paddle.nn.CrossEntropyLoss` 计算中就是调用  `paddle.nn.functional.cross_entropy` ，二者输入变量的维度应该一致才对，那么你们代码中计算交叉熵的时候类别的维度到底是放在第二维还是最后一维？

参考文档连接 [CrossEntropyLoss](https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/nn/layer/loss/CrossEntropyLoss_cn.html#crossentropyloss), [cross_entropy](https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/nn/functional/loss/cross_entropy_cn.html#canshu)

**Comment by huangjun12** [comment]
好的，文档确实有问题，这边安排修复，谢谢反馈~

**Comment by TCChenlong** [comment]
hi，您好，这两个文档是有一些矛盾的地方，是cross_entropy有一些问题，目前我们已经修改完毕了，等更新上线后，就会一致了，感谢您的反馈~
